src/WebUI/WorkLoads/OtherWorkloadsTable.tsx

```diff
@@ -229,7 +242,12 @@ export class OtherWorkloads extends BaseComponent<IOtherWorkloadsProperties, IOt
     }
 
     private static _renderPodsCountCell(rowIndex: number, columnIndex: number, tableColumn: ITableColumn<ISetWorkloadTypeItem>, workload: ISetWorkloadTypeItem): JSX.Element {
-        const { statusProps, pods, podsTooltip } = Utils.getPodsStatusProps(workload.currentPodCount, workload.desiredPodCount);
+        let { statusProps, pods, podsTooltip } = Utils.getPodsStatusProps(workload.currentPodCount, workload.desiredPodCount);
+        if (workload.kind === SelectedItemKeys.OrphanPodKey) {
```

Comment:
we should not special handle, we should simply show, all pods are running or 1pod is not ready etc message.